RELEASE.md

```diff
@@ -40,5 +40,6 @@ To perform a release, you should:
 6. Wait for the build to finish, and at the end you should see a merge request link to the managed-tenant repo.
 7. Review the MR (and make any additional changes if required). Ping reviewers on the PR once it's ready. 
 8. When the final release is done, make sure close the merge blocker issue on Github to allow Prow to start merging PRs back to the release branch.
+9. If this is a patch release and a new ClusterServiceVersion (CSV) is generated, please make sure cherry-pick the CSV changes back to the master branch. This needs to be done manually for now (you can add `/cherry-pick` comment and let Tide do it for you) and we will add automation in the future. 
 
```

Comment:
It's unlikely that we will actually be able to cherry-pick a single commit or merge since it will be built up over multiple commits on the release branch. I think you will need to checkout the released CSV onto master instead and submit a PR seperatly, all of which needs to happen before you create the next minor release off master:

```
git checkout upstream/release-v2.1 -- deploy/olm-catalog/integreatly-operator/
```

Reply:
@mikenairn I have updated the comment now and added your suggestion as an example here.